libraries/core-react/src/hooks/usePopper.ts

```diff
@@ -1,25 +1,22 @@
 import type { CSSProperties } from 'react'
 import { usePopper as reactPopper } from 'react-popper'
 
-export const PlacementValues = [
```

Comment:
If the props table in Popover still works without this, you can remove the argTypes from `Popover.stories`:

```
argTypes: {
    placement: {
      control: {
        type: 'select',
        options: PlacementValues,
        defaultValue: 'auto',
      },
    },
  },
```

Reply:
hmmm, I forgot to test the stories. 🙈 Let me check it out!

Reply:
Removed the `argsTypes` as it works without it now @pomfrida  :)